website/docs/docs/on-premises/scim.md

```diff
@@ -0,0 +1,284 @@
+---
+title: On-Premises SCIM API
+---
+
+The Pactflow SCIM API can be added as a façade to your on-premises Pactflow instance. It runs in its own
+docker container and access the Pactflow APIs using a system account API token.
+
+For details on the Pactflow SCIM API, refer to the [main SCIM documentation](/docs/scim/main).
+
+## Installation
+
+### First
+
+Before running the SCIM API, you must have a correctly running Pactflow instance. Follow the 
+[Pactflow On-Premises installation instructions](/docs/on-premises/installation/checklist).
+
+### Then
+
+* Configure the SCIM Docker image to be pulled from Quay. Follow the [Docker image registry instructions](/docs/on-premises/docker-image-registry)
+ to get access to the Quay registry and then run `docker pull quay.io/pactflow/scim-api` to pull down the latest SCIM API image.
+* You can then deploy the docker container to your docker orchestration service (AWS ECS, K8, etc.). You need to provide the external URL of
+your Pactflow instance as the `PACTFLOW_URL` environment variable to the running container.
+
+**NOTE: The SCIM API has to access your Pactflow instance using the external address.** This is due to Pactflow using
+the HAL media format which uses embedded links in the responses.
```

Comment:
If they want to use an internal address, they can add it to the `PACTFLOW_BASE_URLS` environment variable. Then the HAL browser will work correctly. eg. `PACTFLOW_BASE_URL="https://pactflow.external.mycompany.com https://pactflow.internal"` Which ever address is used to access the API will be returned in the HAL browser links.